libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -52,14 +52,22 @@ const AP_Param::GroupInfo AP_LandingGear::var_info[] = {
     // @Description: pin number to use for feedback of weight on wheels condition. If set to -1 then don't use a pin flag for this, otherwise this is a pin number.
     // @Values: -1:Disabled,50:PX4 AUX1,51:PX4 AUX2,52:PX4 AUX3,53:PX4 AUX4,54:PX4 AUX5,55:PX4 AUX6
     // @User: Standard
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    AP_GROUPINFO("WOW_PIN", 5, AP_LandingGear, _pin_weight_on_wheels, 8),
+#else
```

Comment:
Instead of this, use a macro, defined at the top of the file, just for the value.